src/admin/components/NavBar/index.js

```diff
@@ -2,16 +2,19 @@ import React, { PropTypes } from 'react';
 
 import {
   NavBar,
+  NavBarButton,
   NavBarLink,
 } from 'core/components/NavBar';
 
-const AdminNavBar = () => (
+const AdminNavBar = ({ authenticated, logOut }) => (
   <NavBar>
     <NavBarLink to="/search">Search</NavBarLink>
+    {authenticated ? <NavBarButton onClick={logOut}>Log out</NavBarButton> : null}
   </NavBar>
 );
 AdminNavBar.propTypes = {
-  logout: PropTypes.func.isRequired,
+  logOut: PropTypes.func.isRequired,
+  authenticated: PropTypes.bool.isRequired,
```

Comment:
I prefer it without but that could be my old ruby days coming through. I'll change it.